editor/editor_file_dialog.cpp

```diff
@@ -453,6 +463,8 @@ void EditorFileDialog::_item_selected(int p_item) {
 
 		file->set_text(d["name"]);
 		_request_single_thumbnail(get_current_dir().plus_file(get_current_file()));
+	} else if (mode == MODE_OPEN_DIR) {
+		get_ok()->set_text(RTR("Select this Folder"));
```

Comment:
In my opinion, I'd do `"Select Folder"` like how Windows does with its prompts. Perhaps that's down to personal opinion, though.

On that note though, I don't see the need to switch the text when a item is selected. Why not just keep it as the same text in both instances?

Barring all of that though, I'd at least recommend this be changed to `"Select This Folder"` to stay consistent.

Reply:
There's a difference to avoid confusion if you're selecting the folder you're currently in, or the select folder by the cursor.

About the "this", I kept as lowercase because it's also like that in the FileDialog, but I can capitalize both,